### ISSUE TITLE
[Bug] Errors on examples

### ISSUE THREAD

**Issue by jeromelachaud** [solidjs/solid-start#338]
Got ths error on `auth` and `bare` examples:
```
[plugin:solid] /node_modules/@solidjs/router/dist/components.jsx: transformCondition is not defined
/node_modules/@solidjs/router/dist/components.jsx
```

**Comment by croutonn** [comment]
It seems that the `babel-plugin-jsx-dom-expressions` update is now causing the problem.
Setting the version to `0.34.13` in the overrides field of package.json works fine.

```json
{
  "overrides": {
    "babel-plugin-jsx-dom-expressions": "0.34.13"
  }
}
```

**Comment by ryansolid** [comment]
Yeah missed a code path and was missing an import.. Should be fixed in 0.35.1. You can get that by blowing out your node modules and locks I imagine.

**Comment by jeromelachaud** [comment]
Yep 👍 Works now.
